xc7/archs/artix7/devices/xc7a50t-basys3-x1y0-roi-virt/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_xc7_device_define_type(
+  ARCH artix7
+  DEVICE xc7a50t-basys3-x1y0
+  ROI_PART xc7a35tcpg236-1
+  ROI_DIR ${PRJXRAY_DB_DIR}/artix7/harness/basys3/x1y0
```

Comment:
This harness is not part of prjxray-db.  Given that it isn't actually generated from Vivado, should this just be checked in and be part of symbiflow-arch-defs?